app/src/main/java/org/greenstand/android/TreeTracker/fragments/NewTreeFragment.kt

```diff
@@ -225,6 +233,17 @@ class NewTreeFragment : androidx.fragment.app.Fragment(), OnClickListener, Activ
                     }
 
                     setPic()
+                    if (MainActivity.mImageQuality < FOCUS_THRESHOLD) {

```

Comment:
Instead of adding a new property into the MainActivty, can we instead pass back the image quality value using onActivityResult?